libkernelflinger/security.c

```diff
@@ -77,6 +77,7 @@ union android_version {
      } __attribute__((packed)) split;
 };
 
+static struct rot_data_t rot_data = {0};
```

Comment:
remove `= {0}`.  This is unnecessary.  This is a static variable => per the C documentation, it is initialized to zero.